sw/device/silicon_creator/lib/boot_svc/boot_svc_empty.h

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_EMPTY_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_EMPTY_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_header.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  kBootSvcEmptyType = 0xb4594546,
```

Comment:
Out of curiosity, how was this message identifier selected? Should we document it?

Perhaps we should define all message identifiers centrally in boot_svc_header.h rather than in individual messages' header files.

Reply:
using the sparse_fsm script, I just requested a number with a HW of at least 11.

I was planning to move these to boot_svc_msg.h that I'm planning to add when I work on the next task.